app/picture.py

```diff
@@ -0,0 +1,55 @@
+"""Picture module."""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from PIL import Image
+
+if TYPE_CHECKING:
+    from tile import Tile
+
+
+class Picture:
+    """Picture class."""
+
+    def __init__(self) -> None:
+        self.image: Image.Image | None = None
+        self.is_solved = False
+        self.tiles: dict[int:Tile] = {}  # init so that autocomplete works
```

Comment:
The only reason this hasn't caused a crash is because of the `__future__` `annotations` import. This is not valid syntax. Generic arguments are split with commas.
```suggestion
        self.tiles: dict[int, Tile] = {}
```